### PR TITLE
Update Instance segmentation Tutorial

### PR DESCRIPTION
**Pull request by ytzfhqs**
New tutorial for fine-tuning the mask_rcnn model

## Motivation
The old Instance segmentation Tutorial documentation is no longer suitable for the current version.As described in issue [#10135](https://github.com/open-mmlab/mmdetection/issues/10135), using the old tutorial documentation results in a ``cannot import name 'build_dataset' from 'mmdet.datasets'`` error!

## Modification

Created a new example Instance segmentation tutorial document to fine-tune the mask_rcnn model on the ballon dataset.And a more detailed explanation of the config file modification.The document works well on the colab platform.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects, like MMDet or MMPreTrain.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

**Comment by hhaAndroid** [comment]
@ytzfhqs Thank you very much. May I ask if you could directly modify the `demo/MMDet_InstanceSeg_Tutorial.ipynb` file instead of creating a new file?

**Comment by ytzfhqs** [comment]
Sure. I've removed the old example Instance segmentation tutorial and aligned the new tutorial filename with the old tutorial filename

**Comment by ytzfhqs** [comment]
@hhaAndroid ，Sure. I've just made a change based on the old tutorial file.

I have previously written a tutorial file for fine tuning the RTMDet Segmentation model, is that tutorial file needed at this stage? If so I would like to add the RTMDet Segmentation model tutorial to the project file.

**Comment by OpenMMLab-Assistant-004** [comment]
Hi @ytzfhqs,

We'd like to express our appreciation for your valuable contributions to the mmdetection. Your efforts have significantly aided in enhancing the project's quality.
It is our pleasure to invite you to join our community thorugh Discord_Special Interest Group (SIG) channel. This is a great place to share your experiences, discuss ideas, and connect with other like-minded people. To become a part of the SIG channel, send a message to the moderator, OpenMMLab, briefly introduce yourself and mention your open-source contributions in the #introductions channel. Our team will gladly facilitate your entry. We eagerly await your presence. Please follow this link to join us: ​https://discord.gg/UjgXkPWNqA.

If you're on WeChat, we'd also love for you to join our community there. Just add our assistant using the WeChat ID: openmmlabwx. When sending the friend request, remember to include the remark "mmsig + Github ID".

Thanks again for your awesome contribution, and we're excited to have you as part of our community!

**Comment by hhaAndroid** [comment]
> @hhaAndroid ，Sure. I've just made a change based on the old tutorial file.
> 
> I have previously written a tutorial file for fine tuning the RTMDet Segmentation model, is that tutorial file needed at this stage? If so I would like to add the RTMDet Segmentation model tutorial to the project file.

Is there any difference between what you wrote and the current tutorial? If you think there is a significant difference, you can submit a pull request.

**Comment by ytzfhqs** [comment]
> > @hhaAndroid ，Sure. I've just made a change based on the old tutorial file.
> > I have previously written a tutorial file for fine tuning the RTMDet Segmentation model, is that tutorial file needed at this stage? If so I would like to add the RTMDet Segmentation model tutorial to the project file.
> 
> Is there any difference between what you wrote and the current tutorial? If you think there is a significant difference, you can submit a pull request.

Well, I don't think there's much that's different, so there's no point in doing duplicate work, I'd prefer this PR to be Merging

**Comment by ashikMostofaTonmoy** [comment]
I think it is high time you guys merge this pull request or update the code base.

**Comment by hhaAndroid** [comment]
@ytzfhqs Can you make changes to the lint? We will merge it.

**Comment by ytzfhqs** [comment]
@hhaAndroid Of course, I've just completed that task.

**Comment by rbli-john** [comment]
> @hhaAndroid ，Sure. I've just made a change based on the old tutorial file.
> 
> I have previously written a tutorial file for fine tuning the RTMDet Segmentation model, is that tutorial file needed at this stage? If so I would like to add the RTMDet Segmentation model tutorial to the project file.

@ytzfhqs Hi, where can I find the tutorial file for fine tuning the RTMDet Segmentation model? I do not find it in the main branch.

**Comment by ytzfhqs** [comment]
> > @hhaAndroid ，Sure. I've just made a change based on the old tutorial file.
> > I have previously written a tutorial file for fine tuning the RTMDet Segmentation model, is that tutorial file needed at this stage? If so I would like to add the RTMDet Segmentation model tutorial to the project file.
> 
> @ytzfhqs Hi, where can I find the tutorial file for fine tuning the RTMDet Segmentation model? I do not find it in the main branch.

@rbli-john I did not propose PR as I considered that adding a tutorial on RTMDet model would lead to redundancy in the target detection tutorial. but I have done a tutorial on RTMDet model on the colab platform and [shared it](https://colab.research.google.com/drive/1rd57-38NeZCkOcG6fPn9xjbfYD5nggFd?usp=sharing) with you in the hope that it will help you!

**Comment by rbli-john** [comment]
> > > @hhaAndroid ，Sure. I've just made a change based on the old tutorial file.
> > > I have previously written a tutorial file for fine tuning the RTMDet Segmentation model, is that tutorial file needed at this stage? If so I would like to add the RTMDet Segmentation model tutorial to the project file.
> > 
> > 
> > @ytzfhqs Hi, where can I find the tutorial file for fine tuning the RTMDet Segmentation model? I do not find it in the main branch.
> 
> @rbli-john I did not propose PR as I considered that adding a tutorial on RTMDet model would lead to redundancy in the target detection tutorial. but I have done a tutorial on RTMDet model on the colab platform and [shared it](https://colab.research.google.com/drive/1rd57-38NeZCkOcG6fPn9xjbfYD5nggFd?usp=sharing) with you in the hope that it will help you!

@ytzfhqs Thank you, that would be helpful. But when I click the link you shared, it said that I need to grant the access permission. I've send the request, could you please give the permission to me. Thanks.

**Comment by ytzfhqs** [comment]
@rbli-john Sorry, I forgot to change the permissions.

**Comment by rbli-john** [comment]
> @rbli-john Sorry, I forgot to change the permissions.[Git it](https://colab.research.google.com/drive/1rd57-38NeZCkOcG6fPn9xjbfYD5nggFd?usp=sharing)

@ytzfhqs  I can access now, thank you.

**Comment by rbli-john** [comment]
> @rbli-john Sorry, I forgot to change the permissions.[Git it](https://colab.research.google.com/drive/1rd57-38NeZCkOcG6fPn9xjbfYD5nggFd?usp=sharing)

@ytzfhqs Hi, I used the notebook in your link above to finetune the RTM-det on ballon datasets. I do not change any code. However, after 130 epochs training, I can only get mask mAP about 61%, which is much lower than that of Mask-RCNN which is used in MMDet_InstanceSeg_Tutorial.ipynb (https://github.com/open-mmlab/mmdetection/blob/dev-3.x/demo/MMDet_InstanceSeg_Tutorial.ipynb). And, I used the fineturned weight to do prediction on the same image in the notebook, which get a large area of false positive. See screen below:
image

May I ask:
1. What mask mAP you can got when running your RTMDet Segmentation tutorial?
2. Do I miss something (so that I do not get a good result)?

**Comment by rbli-john** [comment]
> > @rbli-john Sorry, I forgot to change the permissions.[Git it](https://colab.research.google.com/drive/1rd57-38NeZCkOcG6fPn9xjbfYD5nggFd?usp=sharing)
> 
> @ytzfhqs Hi, I used the notebook in your link above to finetune the RTM-det on ballon datasets. I do not change any code. However, after 130 epochs training, I can only get mask mAP about 0.61, which is much lower than that of Mask-RCNN (about 0.81) used in MMDet_InstanceSeg_Tutorial.ipynb (https://github.com/open-mmlab/mmdetection/blob/dev-3.x/demo/MMDet_InstanceSeg_Tutorial.ipynb). Additionally, I used the fineturned weight to do prediction on the same image in the notebook, but get a large area of false positive. See screen below: image
> 
> May I ask:
> 
> 1. What mask mAP you can got when running your RTMDet Segmentation tutorial?
> 2. Do I miss something (so that I do not get a good result)?

@ytzfhqs hi, do you have any answer (or any comments) on my question above? Very appreciate if you can help.
